packages/resources/src/util/nodeBuilder.ts

```diff
@@ -220,7 +222,8 @@ export function builder(builderProps: BuilderProps): BuilderOutput {
   let outCode, outHandler;
   if (bundle) {
     outCode = lambda.Code.fromAsset(buildPath);
-    outHandler = path.basename(handler);
+    outHandler = path.join(path.dirname(entryPath), path.basename(handler));
```

Comment:
Lambda needs the handler path in posix format ie. "/" this slash. We need to test deploying on Windows environment.